content/books/arcanea-creator-principles/chapter-01-the-seven-principles.md

```
@@ -256,22 +256,22 @@ The Creator Principles integrate with:
 
 ## The Scrolls
 
-### [Scroll I: The Partnership Principle](chapters/01-partnership-principle.md)
+### [Scroll I: The Partnership Principle](/books/arcanea-creator-principles/chapter-02-the-partnership-principle)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)